Feature/separate temps

tdorsey opened the pull request:
This adds a separate panel to distinguish between disk temperature and airflow temperature

tdorsey committed:
Add dashboard json

tdorsey committed:
Separate airflow and disk temperatures

Airflow temperatures and disk temperatures are now displayed in separate
panels side by side

fixes [#1](https://github.com/galexrt/docker-node_exporter-smartmon/issues/1)

galexrt commented:
@tdorsey Please update / edit the existing `grafana/smartmon-textfile.json` dashboard file.

Additionally if you have time to add a screenshot of the updated dashboard to the README (the screenshot should be added to a new `media/` folder) that would be awesome.

tdorsey committed:
Make requested changes

- Update README with new screenshot
- Edit existing dashboard json
- Add media folder with screenshot of new Airflow/Disk Temperature Panel

tdorsey commented:
Any particular place for the grafana version? I didn't see one listed already. Also, the dashboard format has apparently changed a ton, may want to give it a spin yourself to make sure its still loadable

tdorsey committed:
Fix screenshot path

galexrt commented:
@tdorsey I would say in the https://github.com/galexrt/docker-node_exporter-smartmon#grafana-dashboard "Grafana Dashboard" section as an added sentence.

I will check in the next few hours.

galexrt commented on README.md:
```diff
@@ -21,6 +21,9 @@ The entrypoint script is putting the output into the directory `/var/lib/node_ex
 A Grafana dashboard for viewing the exported metrics can be found on [Grafana.com - Smartmon Textfile by Galexrt](https://grafana.com/dashboards/3992) and in the [`grafana/`](grafana/) directory.
 
+### Screenshots  
+    ![Disk and Airflow Temperatures](media/disk-airflow-temps.png?raw=true "Disk and Airflow Temps")
```
```suggestion
![Disk and Airflow Temperatures](media/disk-airflow-temps.png?raw=true "Disk and Airflow Temps")
```

galexrt commented:
I can't check if it works because the `smartmon.sh` script doesn't seem to pickup my NVMe drives..

I have imported the dashboard and fixed the issues I saw with it (attached as a txt as JSON isn't allowed to be uploaded): [Smartmon Textfile-1603663177150.json.txt](https://github.com/galexrt/docker-node_exporter-smartmon/files/5436031/Smartmon.Textfile-1603663177150.json.txt)

I have also gone ahead and updated the `entrypoint.sh` and image to simply be a wrapper around the node_exporter textfile exporter scripts repo, so any script from that repo can be run with easy (default is still the `smartmon.sh` script to be backwards compatible).

tdorsey committed:
Fix screenshot display

Co-authored-by: Alexander Trost <Galexrt@googlemail.com>

tdorsey committed:
Include suggestions from review

tdorsey committed:
Add grafana and dashboard schema versions

galexrt commented on README.md:
```diff
@@ -21,6 +21,11 @@ The entrypoint script is putting the output into the directory `/var/lib/node_ex
 
 A Grafana dashboard for viewing the exported metrics can be found on [Grafana.com - Smartmon Textfile by Galexrt](https://grafana.com/dashboards/3992) and in the [`grafana/`](grafana/) directory.
 
+Tested against Grafana version 6.1.6 and schema version 18
```
```suggestion
Tested against Grafana version `6.1.6` and schema version `18`.
```

galexrt commented:
One last comment, then I'll go ahead merge the PR :slightly_smiling_face:
